### DISCUSSION TITLE
VKPSOC fail to load design on Libero lauched ModelSim

### DISCUSSION THREAD

**Discussion by wxtang123** [polarfire-soc/polarfire-soc-documentation#414] [Reference designs, FPGA and development kits]
I try to run simulation before synthesis, 
but a error report as follow occured.
![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/167262881/82309144-6599-4f71-b224-b40e76403c18)
The IMAGE_SCALAR is v4.1.0 ,
what I may miss to run the baisc simulation on top design?

**Comment by sc-mchp** [comment]
Can you please provide steps to reproducers this problem
